components/common/ProfileInfoCard.tsx

```diff
@@ -0,0 +1,34 @@
+import { Flex, Box, Image, Badge, Text, AspectRatio } from "@chakra-ui/react";
+
+type CardProps = {
```

Comment:
Pls fix the lint warnings coming in console on yarn-build

<img width="1025" alt="image" src="https://user-images.githubusercontent.com/7558499/168874675-3edbf951-0b2c-45bb-a40f-61d0fa55e9b9.png">